client/ConnorsNotes.md

```diff
@@ -0,0 +1,9 @@
+Removed style={eventstyles} from Event.js => events.css
+
+centered search bar at 80% viewport width
+
+Moved <h1>Event Manager<h1> up and gave className="header-header"
+
+Centered <h1><h1>
+
+added footer.css
```

Comment:
These should be your commit messages instead of "v1" or "changes". These aren't part of the application or its deployment, so they should be tracked in metadata, i.e. git log.

Reply:
yeah, sorry. First open source project. I knew I need to record the changed but didnt think about multiple commits